bin/cml/runner.js

```diff
@@ -432,7 +432,6 @@ exports.handler = async (opts) => {
     await run(opts);
   } catch (error) {
     await shutdown({ ...opts, error });
-    throw error;
```

Comment:
Why not throw the error so it's displayed on the command-line output?

Reply:
Are we ever reaching that? if shutdown succeeds process will terminate

Reply:
Good point! 🤦🏼‍♂️ Sounds like more “logic to revisit”, but makes sense.

Reply:
I was there because shutdown was meant to throw errors originally but shutdown is now bulletproof

Reply:
> Good point! 🤦🏼‍♂️ Sounds like more “logic to revisit”, but makes sense.

Im doing that in small PRs to not have a monolithic one